L1Trigger/TrackFindingTracklet/python/Tracklet_cfi.py

```diff
@@ -23,7 +23,9 @@
                                                TrackQuality = cms.bool(True),
                                                TrackQualityPSet = cms.PSet(TrackQualityParams),
                                                Fakefit = cms.bool(False), # True causes Tracklet reco to output TTTracks before DR & KF
-                                               StoreTrackBuilderOutput = cms.bool(False) # if True EDProducts for TrackBuilder tracks and stubs will be filled
+                                               StoreTrackBuilderOutput = cms.bool(False), # if True EDProducts for TrackBuilder tracks and stubs will be filled
+                                               RemovalType = cms.string("merge"),
```

Comment:
Add small comments to end of L27-28: "Duplicate track removal"; "Allow tracklet tracks multiple stubs per layer".

Reply:
done